### PR TITLE
[6.3] Port external capsule + RFT

### PR DESCRIPTION
**Pull request by ldjebran**
test log: http://pastebin.test.redhat.com/491756
```console
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test -v tests/foreman/cli/test_contentview.py::ContentViewTestCase::test_positive_remove_cv_version_from_multi_env_capsule_scenario
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.0.7, py-1.4.32, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.0, cov-2.4.0
collected 73 items 
2017-06-07 10:13:44 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings


tests/foreman/cli/test_contentview.py::ContentViewTestCase::test_positive_remove_cv_version_from_multi_env_capsule_scenario <- robottelo/decorators/__init__.py PASSED

============================================= 1 passed in 4234.82 seconds ==============================================
```

### CODE REVIEW

**Comment by ldjebran** [comment]
@kbidarkar, this PR needs your attention as we will need the configuration to be inserted the same way as for capsule in 6.2.z

**Comment by abalakh** [comment]
@ldjebran before we merge it, are CI changes in place already, will `[capsule]` be uncommented? Can't see any linked robottelo-ci or automation-tools PR.

**Comment by ldjebran** [comment]
@abalakh , yes we have to check with @kbidarkar for this. these PRs have to be merged the same time

**Comment by ldjebran** [comment]
@abalakh nothing special, the same code, only removed compatibility with rhel6 and special repo selection in master, and resolved some problem with the capsule installer cmd

**Comment by ldjebran** [comment]
@abalakh CI changes already in place

File: robottelo.properties.sample [hunk 1]
```diff
@@ -383,3 +383,22 @@ port_range=9091, 14999
 # image_name: Image name on the external provider
 # image_name=
 
+# section for real capsule
+# [capsule]
+# to setup a real capsule we need the capsule virtual machine hostname
+# resolvable.
+# we should register a hostname and get a hash value as described in
+# https://mojo.redhat.com/docs/DOC-13143, that we should use on the stage of
```
Comment (renzon) [inline robottelo.properties.sample]: I think this link should not be public

**Comment by ldjebran** [comment]
@renzon the change done, it requires no action from @kbidarkar, he has already done all the necessary changes

**Pull request merged**